modules/schema/src/main/java/org/apache/ignite/internal/schema/configuration/SchemaConfigurationConverter.java

```diff
@@ -589,7 +589,7 @@ else if (cls == Float.class)
         else if (cls == Double.class)
             return ColumnType.DOUBLE;
 
-        // Temporal types.
+        // Time interval types.
```

Comment:
We don't support interval types.
These are temporal, not interval.

Reply:
change for: Date time types. hope it`s more clear.